sdcm/cluster.py

```diff
@@ -2117,7 +2117,9 @@ def web_install_scylla(self, scylla_version: Optional[str] = None) -> None:
         else:
             raise Exception("Scylla version for web install isn't identified")
 
-        self.remoter.run(f"curl -sSf get.scylladb.com/server | sudo bash -s -- --scylla-version {version}")
+        product_type = '--scylla-product scylla-enterprise' if self.is_enterprise else ''
```

Comment:
you have a chicken and an egg problem here, `is_enterprise` is only working once scylla is installed

Reply:
I would recommend adding a new utility function to guess if it's enterprise 
base on `'enterprise' in self.params('scylla_repo')` or  `'enterprise' in os.environ.get('JOB_NAME')`  it's a bit clunky, but might do the trick.

Reply:
yep, figured that out... just thought that, for now, i can check the "version" format, and everything that will be `20xx.1` is enterprise? does that even make sense?

Reply:
what do you mean version format ? you don't have a version yet anywhere
the only thing you are triggered with is `scylla_repo`, since it triggered like all other artifact tests

Reply:
we do have some kind of version:
`--scylla-version nightly-2022.1`

it comes from `if match := re.match(r'\D*(\d+\.\d+)', scylla_version or git_branch):`

so in the bottom line, this "version" will give the me format i need to search, as i just did (and i have a build running to check it)
LMK what you think @fruch